internal/config/common_test.go

```diff
@@ -85,3 +87,31 @@ func TestGenerateAddress(t *testing.T) {
 		})
 	}
 }
+
+func CreateTempConfigFile(content string, t *testing.T) (string, func()) {
```

Comment:
NIT: This doesn't need to be exported

Reply:
It does, as it's used in `config_test`

Reply:
Although, some reshuffling means we can move things into their natural place. Will sort this out.